src/main/java/io/jenkins/plugins/endpoints/PluginEndpoint.java

```diff
@@ -55,4 +66,27 @@ public Plugin getPlugin(@PathParam("name") String name) {
     }
   }
 
+  /**
+   * <p>Get issues</p>
+   *
+   * @param name The plugin to retrieve
+   * @return Matching issues
+   */
+  @Path("/issues/open")
+  @GET
+  public JiraIssues getIssue(@PathParam("name") String name) {
+    try {
+      final Plugin plugin = datastoreService.getPlugin(name);
+      if (plugin == null) {
+        throw new WebApplicationException(Response.Status.NOT_FOUND);
+      }
+      JiraIssues jiraIssues = new HttpClientJiraIssues(configurationService).getIssues(plugin.getName());
+      // jiraIssues.issues = searchResult.getIssues();
```

Comment:
?

Reply:
Ack, this got reverted, I had cleaned it up.